snake/common/variants/variant_mod_snake.py

```diff
@@ -113,24 +113,28 @@ def getBamForHeaderUpdate(wildcards):
     shell:
         '{config[tools][bcftools][call]} index {params.params} {input.vcf}'
         
+if not 'CONCATVARSCANIN' in globals():
+	CONCATVARSCANIN = VARSCANSOMATICOUT
+if not 'CONCATVARSCANOUT' in globals():
+	CONCATVARSCANOUT = OUTDIR + 'variants/varscan_somatic/concatenated/'
 rule bcftoolsConcat:
     input:
-        vcfIndel = '{sample}.indel_fullHeader.vcf.gz',
-        vcfSnp = '{sample}.snp_fullHeader.vcf.gz',
-        vcfIndelIndex = '{sample}.indel_fullHeader.vcf.gz.csi',
-        vcfSnpIndex = '{sample}.snp_fullHeader.vcf.gz.csi'
+        vcfIndel = CONCATVARSCANIN + '{sample}.indel_fullHeader.vcf.gz',
+        vcfSnp = CONCATVARSCANIN + '{sample}.snp_fullHeader.vcf.gz',
+        vcfIndelIndex = CONCATVARSCANIN + '{sample}.indel_fullHeader.vcf.gz.csi',
+        vcfSnpIndex = CONCATVARSCANIN + '{sample}.snp_fullHeader.vcf.gz.csi'
     output:
-        vcf = '{sample}_complete.vcf'
+        vcf = CONCATVARSCANOUT + '{sample}.vcf'
     params:
-        lsfoutfile = '{sample}_fullHeader.vcf.lsfout.log',
-        lsferrfile = '{sample}_fullHeader.vcf.lsferr.log',
+        lsfoutfile = '{sample}.concat.vcf.lsfout.log',
```

Comment:
please remove 'concat'
```suggestion
        lsfoutfile = CONCATVARSCANOUT +  '{sample}.vcf.lsfout.log',
```